serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -253,6 +259,19 @@ link:{quarkus_k8s_deploy_url}[link].
 ====
 --
 
+.Build and Deploy your workflow application
+[source,shell]
+----
+quarkus build -- -Pcontainer -DskipTests \
+  -Dquarkus.container-image.push \
```

Comment:
In the previous step we declare `quarkus.container-image.push=false`, what do we set here?